sql/moz-fx-data-shared-prod/firefox_accounts_derived/docker_fxa_customs_sanitized_v2/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+friendly_name: Docker Fxa Customs Sanitized
+description: |-
+  Sanitized version of customs events from GCP based FxA service stderr table.
+  PII is hashed.
```

Comment:
I'd suggest mentioning the date range when the GCP migration started and ended (though given the v1 ETL is still pulling data maybe the end is still a bit questionable).

And I'd also suggest updating the description for the v1 table to clarify it only contains logs from when it was running in AWS, and the date range when the GCP migration started and ended.